sdk/core/core-rest-pipeline/test/tracingPolicy.spec.ts

```diff
@@ -154,7 +154,7 @@ describe("tracingPolicy", function () {
 
     const createdSpan = activeInstrumenter.lastSpanCreated;
     assert.exists(createdSpan);
-    const mockSpan = createdSpan!;
+    const mockSpan = createdSpan;
```

Comment:
Always good to see these assertions go away :+1:

I'm guessing vitest has a better type for the assert functions that allowed this?

Reply:
Yea! I dont know if the chai types were improved (vitest uses chai I think?) or if that's vitest but the type of `assert.exists` is now

```ts
(method) Chai.Assert.exists<MockSpan | undefined>(value: MockSpan | undefined, message?: string | undefined): asserts value is MockSpan (+1 overload)
```

Which is really nice!